editor/plugins/node_3d_editor_plugin.h

```diff
@@ -848,6 +849,8 @@ class Node3DEditor : public VBoxContainer {
 	bool is_gizmo_visible() const;
 
 	ToolMode get_tool_mode() const { return tool_mode; }
+	ToolMode get_prev_tool_mode() const { return prev_tool_mode; }
+	void set_prev_tool_mode() { prev_tool_mode = tool_mode; }
```

Comment:
I'd call it something else, `set` implies a setter, with a manual argument, perhaps `update_prev_tool_mode` or similar

Reply:
Good point